modules/nodes-nodes-working-setting-booleans.adoc

```diff
@@ -0,0 +1,56 @@
+[id="nodes-nodes-working-setting-booleans"]
+
+= Setting SELinux booleans
+
+{product-title} allows you to enable and disable a SELinux boolean on a RHEL CoreOC node. The following procedure explains how to modify SELinux booleans on nodes using the MachineConfigOperator (MCO). The `container_manage_cgroup` is the example sebool in this procedure. You can modify this value to whichever sebool you need.
+
+.Prerequisites:
+
+* You have installed the OpenShift CLI (oc).
+
+.Procedure
+
+. Create a new file with the following MachineConfig defined:
```

Comment:
s/ . Create a new file with the following MachineConfig defined: / Create a new file with a `MachineConfig` object like the following example: 

^ Your way may be correct, but I feel like it's missing something (a noun) for clarity. I found my suggestion in the 4.8 docs. 

Alternatively, maybe something like 'Create a new file with the following `MachineConfig` configuration:' but that might be redundant with config/configuration. Maybe 'Create a new file with the following `MachineConfig` object`?

Reply:
Yes, this should be `MachineConfig` object if using the literal value, or "machine object" for using it as a general reference. See "[Object references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references)" guidelines.

I'd suggest something like, "Create a new YAML file that defines the `MachineConfig` object, for example:"